src/applications/disability-benefits/996/components/ReviewDescription.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { selectProfile } from 'platform/user/selectors';
+
+import { formatPhone } from './ContactInformation';
+import { ADDRESS_TYPES, formatAddress } from 'platform/forms/address/helpers';
+
+const ReviewDescription = ({ profile }) => {
+  if (!profile) {
+    return null;
+  }
+
+  const { email, homePhone, mailingAddress } = profile.vapContactInfo;
+  const address = formatAddress(mailingAddress);
+  const isUS = mailingAddress.addressType !== ADDRESS_TYPES.international;
+  const stateOrProvince = isUS ? 'State' : 'Province';
+
+  // Label: formatted value in (design) display order
+  const display = {
+    'Phone number': () =>
+      formatPhone(`${homePhone?.areaCode}${homePhone?.phoneNumber}`),
+    'Email address': () => email.emailAddress,
+    Country: () => address.country,
+    'Street address': () => address.addressLine1,
+    'Line 2': () => address.addressLine2,
+    'Line 3': () => address.addressLine3,
+    City: () => address.city,
+    [stateOrProvince]: () => address.stateOrProvince,
+    'Postal code': () => address.zipOrPostalCode,
```

Comment:
Do you think it'd be better to revert the changes to `formatAddress` then?

Reply:
I went this route so that we could maintain some consistency when formatting addresses. It would be better to use existing code that processes addresses for foreign, domestic and military bases than to roll my own.